File: Java/Medium/WrongAnswer/time-needed-to-rearrange-a-binary-string_5.java
class Solution {
    public int secondsToRemoveOccurrences(String s) {
        
        int count = 0;
        int len = s.length();
        int i = 0;
        while (i < len) {
            if (s.charAt(i) == '0') {
                i++;
            } else {
                i++;
                while (i < len && s.charAt(i) == '1') {
                    i++;
                }
                count++;
            }
        }
        return count;
    }
}

File: C++/Medium/WrongAnswer/time-needed-to-rearrange-a-binary-string_5.cpp
class Solution {
public:
    int secondsToRemoveOccurrences(string s) {
        
        int ans = 0;
        int count = 0;
        for (int i = 0; i < s.length(); i++) {
            if (s[i] == '0') {
                count++;
            }
            else {
                if (count > 0) {
                    ans++;
                    count--;
                }
            }
        }
        return ans;
    }
};